bin/run-in-docker.ps1

```diff
@@ -30,4 +30,4 @@ param (
 )
 
 docker build -t elm-representer .
-docker run --network none --read-only --mount type=bind,src=${InputDirectory},dst=/solution --mount type=bind,src=${OutputDirectory},dst=/output elm-representer $Exercise /solution /output

```

Comment:
I think the --readonly flag should remain for security reasons?

Reply:
I've discussed in the slack about the new setup for containers. There is no more read-only constraint as it proved difficult for other tracks. All modifications are discarded after each run though, just as with docker --rm. For us it enabled simpler code in run.sh. That's why I removed the --readonly param in the run-in-docker scripts

Reply:
By the way, I also did the same in the elm-test-runner repo

Reply:
Yeah I don't think its important, but I was asked to add it in the initial PR ...